.github/workflows/pyrealm_ci.yaml

```diff
@@ -5,8 +5,8 @@ name: Test and build
 on:
   pull_request:
     types: [opened, synchronize, reopened]
-  push:
-    branches: [main, develop]
+  # push:
```

Comment:
Why is this commented out? Shall it be removed?

Reply:
Yes - this was an experiment. Previous CI runs each test twice (once for PR and once for pull). I'm not quite sure _why_ the `pull` event onto `main` or `develop` gets triggered but it doesn't seem necessary?